### PR TITLE
cypress: make annotation specs compaitable with NB

### PR DESCRIPTION
**Pull request by Rash419**
Signed-off-by: Rash419 <rashesh.padia@collabora.com>
Change-Id: I76f70262c13896ccb5bd41f68ac12b337db7f7c2

* Target version: master 

### Summary
I have removed one test from comment scrolling because the behaviour of the comment scrolling has been changed 
Before the comments were scrolling on the individual basics per slide but now they show up in group per slides
hence we don't need that test

### CODE REVIEW

File: cypress_test/integration_tests/common/desktop_helper.js [hunk 1]
```diff
@@ -270,18 +270,41 @@ function deleteImage() {
...
+function insertMultipleComment(docType, numberOfComments = 1, isFirstCall = true) {
+	if (isFirstCall) {
+		var mode = Cypress.env('USER_INTERFACE');
+
+		cy.get('#toolbar-up .w2ui-scroll-right').click();
+
+		if (mode === 'notebookbar') {
+			cy.get('#Insert-tab-label').click();
+
+			(docType === 'writer') ?
```
Comment (lpranam) [inline cypress_test/integration_tests/common/desktop_helper.js:284]: a simple if is better in this case

File: cypress_test/integration_tests/common/desktop_helper.js [hunk 2]
```diff
@@ -270,18 +270,41 @@ function deleteImage() {
 		.should('not.exist');
 }
 
-function insertMultipleComment(numberOfComments) {
-	numberOfComments = numberOfComments || 1;
+// we don't need to scroll if toolbar is already scrolled
+// for the mentioned purpose isFirstCall parameter is used
+function insertMultipleComment(docType, numberOfComments = 1, isFirstCall = true) {
```
Comment (lpranam) [inline cypress_test/integration_tests/common/desktop_helper.js:275]: why to have an extra parameter? instead you could just check if the 'scroll-right' is available then do it straight away!?

File: cypress_test/integration_tests/desktop/impress/annotation_spec.js [hunk 3]
```diff
@@ -13,6 +13,8 @@ describe('Annotation Tests', function() {
 
 		if (Cypress.env('INTEGRATION') === 'nextcloud') {
 			desktopHelper.hideSidebarIfVisible();
+		} else if (Cypress.env('USER_INTERFACE') === 'notebookbar') {
+			cy.get('#ModifyPage').click();
```
Comment (lpranam) [inline cypress_test/integration_tests/desktop/impress/annotation_spec.js:17]: This seems like a bug can you confirm please. Modify page should not work as sidebar button.

**Review by lpranam — CHANGES_REQUESTED** [review]

File: cypress_test/integration_tests/desktop/impress/annotation_spec.js (hunk 3 above)
Comment (Rash419) [reply cypress_test/integration_tests/desktop/impress/annotation_spec.js:17]: I don't think it is a bug : )

**Review by lpranam — APPROVED** [review]

**Pull request merged**